quipucords/tests/scanner/openshift/test_inspect.py

```diff
@@ -115,6 +118,14 @@ def cluster_err(error):
     return OCPCluster(uuid="yet-another-not-uuid", errors={"some-error": error.dict()})
 
 
+@pytest.fixture
+def use_dummy_feature_flag(settings):
```

Comment:
I think the fixture name should make it clear that workloads is enabled.
maybe `workloads_enabled`?

Reply:
Good idea!